### PR TITLE
Feature/oidc10rp allowed resource servers

### PR DESCRIPTION
**Pull request by oharsta**

### CODE REVIEW

File: roles/oidcng/tasks/main.yml [hunk 1]
```diff
@@ -63,6 +63,13 @@
   notify:
     - "restart oidcng"
 
+- name: copy oidcng well-known discovery configuration
+  template:
+    src: "openid-configuration.json.j2"
+    dest: "{{ oidcng_dir }}/openid-configuration.json"
+  notify:
+    - "restart oidcng"
+
```
Comment (quartje) [inline roles/oidcng/tasks/main.yml]: Can this be a reload?

File: roles/oidcng/templates/openid-configuration.json.j2 [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    "http://{{ env }}.surfconext.nl/assurance/loa2",
+    "http://{{ env }}.surfconext.nl/assurance/loa3",
+    "http://{{ env }}.surfconext.nl/assurance/sfo-level2",
+    "http://{{ env }}.surfconext.nl/assurance/sfo-level3"
+  ],
+  "code_challenge_methods_supported": [
+    "plain",
+    "S256"
+  ]
+}
```
Comment (quartje) [inline roles/oidcng/templates/openid-configuration.json.j2]: I reviewed this change in the other PR

File: roles/oidcng/tasks/main.yml (hunk 1 above)
Comment (oharsta) [reply roles/oidcng/tasks/main.yml]: Reload and restart are the same for Spring boot app

File: roles/oidcng/templates/openid-configuration.json.j2 (hunk 2 above)
Comment (oharsta) [reply roles/oidcng/templates/openid-configuration.json.j2]: That's correct. I needed to merge feature branches otherwise the deploy would fail

**Pull request merged**